src/SprykerEco/Zed/Payone/Business/PayoneBusinessFactory.php

```diff
@@ -64,30 +128,287 @@ class PayoneBusinessFactory extends AbstractBusinessFactory
     private $standardParameter;
 
     /**
-     * @return \SprykerEco\Zed\Payone\Business\Payment\PaymentManagerInterface
+     * @return \SprykerEco\Zed\Payone\Business\Payment\MethodSender\PayonePreAuthorizeMethodSenderInterface
      */
-    public function createPaymentManager(): PaymentManagerInterface
+    public function createPreAuthorizeMethodSender(): PayonePreAuthorizeMethodSenderInterface
     {
-        $paymentManager = new PaymentManager(
+        $paymentManager = new PayonePreAuthorizeMethodSender(
```

Comment:
* The temporary variable is not needed here.
* Check similar places.

Reply:
++